### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by matijagaspar**
I updated the versions of `joi`, `core-js`, `mocha` and `temp` to newer versions. I ran all the tests and they pass. 

This gets rid of the warrning regarding joi (#37)

### CODE REVIEW

**Comment by sodatea** [comment]
`joi` is now moved to `@hapi/joi`

**Comment by matijagaspar** [comment]
I will update the pr

**Comment by andrewbruno** [comment]
First of all, great library and thank you.

I was wondering there are any plans to update the release with these updates?

Currently receiving these warnings for devDependency

"wait-on": "^3.2.0"

npm WARN deprecated joi@13.7.0: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated hoek@5.0.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated topo@3.0.3: This module has moved and is now available at @hapi/topo. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
npm WARN deprecated core-js@1.2.7: core-js@<2.6.5 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
npm WARN deprecated hoek@6.1.3: This module has moved and is now available at @hapi/hoek. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.

Currently:

wait-on@3.2.0
  └─┬ joi@13.7.0
         ├── hoek@5.0.4, etc...

If there is anything I can do to help out, please let me know :)

Cheers,
Andrew

**Comment by matijagaspar** [comment]
Updated the `package.json` to use the [@hapi/joi](https://www.npmjs.com/package/@hapi/joi)

**Comment by andrewbruno** [comment]
That's great - Thanks.  Could a new version of wait-on be published? Current at v3.2.0

**Comment by matijagaspar** [comment]
Yes please!

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "6"
   - "8"
   - "10"
```
Comment (paulmelnikow) [inline .travis.yml]: Could you add Node 11 and 12 which are both currently supported?

https://nodejs.org/en/about/releases/

**Comment by danoespinoza** [comment]
@matijagaspar please see the review from @paulmelnikow , thanks for pull-request and new version. Regards.

**Comment by jeffbski** [comment]
Published to npm wait-on@3.3.0  

I updated mocha@6.1.4 to fix a security audit warning.

Thanks for the PR!

**Pull request merged**